contributors/design-proposals/architecture/0000-kep-template.md

```diff
@@ -1,46 +1,185 @@
-[//]: # ( thank you for creating a KEP! )
-[//]: # ( read the  suggested section content: https://github.com/calebamiles/community/blob/propose-kep-template/contributors/design-proposals/architecture/kep-template-instructions.md )
-[//]: # ( replace `Title` with the KEP title ) 
-[//]: # ( replace section content with your amazing proposal )
-[//]: # ( KEP filename should match title, replace spaces with `- `)
-[//]: # ( update table of contents  before merge )
-[//]: # ( remove comments before merge )
-[//]: # ( profit )
-
 # Title
 
+This is the title of the KEP.  Keep it simple and descriptive. A good title can
+help communicate what the KEP is and should be considered as part of any review.
+
+The *filename* for the KEP should include the KEP number along with the title.
+The title should be lowercased and spaces/punctuation should be replaced with
+`-`. As the KEP is approved and an official KEP number is allocated, the file
+should be renamed.
+
+To get started with this template:
+* Make a copy in the appropriate directory.  Name it `draft-YYYYMMDD-my-title.md`.
+* Create a PR in the
+  [`kubernetes/community`](https://github.com/kubernetes/community) repo.
+* Check in early.  View anything marked as a draft as a working document.  By
```

Comment:
It would be useful to indicate minimum requirements for merge.

I assume the proposal still needs to be reviewed prior to merge? There's no good way to use github to review content that has already been merged AFAIK. One needs to open a PR that proposes specific changes.

Reply:
Tweaked it a bit.  Take a look when I push.

As for comments -- I'm thinking we view this as "PRs accepted".  If something doesn't work submit a PR to fix it and discuss in that.